datacenterlight/views.py

```diff
@@ -426,6 +426,10 @@ def get(self, request, *args, **kwargs):
         customer = StripeCustomer.objects.filter(id=stripe_customer_id).first()
         stripe_utils = StripeUtils()
         card_details = stripe_utils.get_card_details(customer.stripe_id, request.session.get('token'))
+        if not card_details.get('response_object') and not card_details.get('paid'):
+            msg = _('Currently its not possible to make payments. Please try later.')
```

Comment:
I think we can obtain the message from the `card_details` object. If I understand it correctly, the error handling correctly takes place in `utils.stripe_utils.handleStripeError` and in case of an error it sets the object to a dictionary containing `error` key whose value is the detailed error message.

Also, its best to add translations there (utils.stripe_utils.handleStripeError). We are missing translations for the error texts there.